docs/getting-started.md

```diff
@@ -0,0 +1,71 @@
+# Getting started
+
+You will need to be running the version of NodeJS set in the `.nvmrc` file in the root of the repository and `Yarn` package manager.
```

Comment:
is this because digital-form-builder requires yarn?

Reply:
Yep